File: fixtures/aspnet-risky/Controllers/ChatController.cs
using Microsoft.AspNetCore.Mvc;

namespace AspNetRisky.Controllers;

[ApiController]
public class ChatController : ControllerBase
{
    [HttpPost("/api/chat")]
    public IActionResult Chat([FromBody] ChatRequest body)
    {
        Console.WriteLine($"Chat request: {body.Message}");
        return Ok(new { text = "response placeholder" });
    }

    [HttpDelete("/api/admin/users")]
    public IActionResult DeleteUser([FromBody] DeleteRequest body)
    {
        Console.WriteLine($"Deleting user: {body.UserId}");
        return Ok(new { ok = true });
    }

    [HttpPost("/api/stripe/webhook")]
    public IActionResult StripeWebhook([FromBody] Dictionary<string, object> event)
    {
        Console.WriteLine($"Webhook event: {event.GetValueOrDefault("type")}");
        return Ok(new { received = true });
    }
}

public class ChatRequest { public string Message { get; set; } = ""; }
public class DeleteRequest { public string UserId { get; set; } = ""; }

File: fixtures/swift-risky/Sources/App/ChatController.swift
import Vapor

func routes(_ app: Application) throws {
    app.post("api", "chat") { req -> Response in
        let body = try req.content.decode([String: String].self)
        print("Chat request: \(body["message"] ?? "")")
        return Response(status: .ok)
    }

    app.delete("api", "admin", "users") { req -> Response in
        let body = try req.content.decode([String: String].self)
        print("Deleting user: \(body["userId"] ?? "")")
        return Response(status: .ok)
    }

    app.post("api", "stripe", "webhook") { req -> Response in
        let event = try req.content.decode([String: Any].self)
        print("Webhook event: \(event["type"] ?? "")")
        return Response(status: .ok)
    }
}
